content/docs/ui/managing-contacts/segmenting-your-contacts.md

```diff
@@ -11,132 +11,64 @@ seo:
 
-<call-out>
+As the traits of your contacts change and you add more contacts that meet the criteria of your segment over time, your segments will dynamically update to reflect these updates. For example, a segment with the criteria “lives in Denver” or “is under 30 years old” will evolve as a contact’s address changes and they grow older. 
 
-You can create up to 200 segments.
+## Creating a Segment
 
-</call-out>
+You can create a new segment with multiple conditions based on the information you have stored about your contacts, and based on some of the information SendGrid has stored about your Marketing Campaigns.
 
 *To create a segment:*
 
 1. Navigate to **Marketing** and then select **Contacts**.
-2. Click **Create Segment**.
-3. Choose **Segment all contacts** or **Segment an existing list**.
-4. Enter a *Segment Name*.
-5. To segment an existing list, select a list from the *Select List to Segment* drop-down.
-6. From the *Condition* drop-down, choose the desired condition or custom field.
-7. From the *Criteria* drop-down, choose from a list of available operators based on the chosen condition.
-
-
-For example, to create a segment of recipients that have clicked on a link in a specific Marketing Campaign:
-
+2. Click on the **Create** dropdown button, and then the **Create Segment** option.
+3. Enter a *Segment Name*.
+4. Choose **Segment all contacts** or **Segment an existing list**.
+5. To segment an existing list, select a list from the *Existing List* drop-down.
+6. In the *targeting* section, start building your condition by choosing a *field* from the drop-down.
+7. Choose from a list of available *operators* based on the chosen field.
+8. Enter the desired *value*.
+9. You can add multiple conditions, and choose to join them with **AND** or **OR**.
+10. Once you are finished adding conditions, click **Save Segment**.
+
+*For example, to create a segment of recipients that live in Denver:*
+ 
 1. Navigate to **Marketing** and then select **Contacts**.
-1. Click **Create Segment**.
+1. Select the **Create** drop-down.
+1. Click **New Segment**.
+1. Enter a Segment Name.
 1. Choose **Segment all contacts** or **Segment an existing list**.
-1. Enter a *Segment Name*.
-1. From the *Condition* drop-down, select **Engagement:Clicks**.
-1. From the *Criteria* drop-down. select **clicked**.
-1. From the last drop-down, select the name of the campaign you want to use for the segment.
+1. From the targeting drop-down, select **City**. 
+1. Select the operator **is**. 
```

Comment:
```suggestion
1. Select the operator **IS**. 
```

Reply:
Not 100% sure

Reply:
I'm matching the UI, so unfortunately, it's lowercase.